lib/jwt_signed_request/key_store.rb

```diff
@@ -2,25 +2,29 @@
 
 module JWTSignedRequest
   class KeyStore
+    def self.find(id)
+      id.nil? ? JWTSignedRequest.key_store : JWTSignedRequest.key_store(id)
```

Comment:
There's a circular dependency here. `JWTSignedRequest` depends on `KeyStore` and `KeyStore` depends on `JWTSignedRequest`. It'd be nice to resolve this.

Reply:
Resolved: 2dabb8b